### DISCUSSION TITLE
Welcome to yotsubee Discussions!

### DISCUSSION THREAD

**Discussion by peteblank** [peteblank/yotsubee#2] [Announcements]
This is for the /g/ame. 
My idea for yotsubee are as follow:
1.yotsubee starts at her house and then she talks to her caretaker , well the guy
2.the guy asks yotsubee to buy some groceries and gives her 100 yeens
3.she goes to the store and buys some stuff(needs inventory)
4.she comes back and the cat runs away
5.she chases the cat around town
6.There ought to be a combat system for casual fighting and a dance dance revolution for a minigame
That's it for now.

**Comment by genericca** [comment]
N

**Comment by peteblank** [comment]
agger hello
